### PR TITLE
VEE Wear

### PR DESCRIPTION
**Pull request by thomasponsmicroej**
Documentation of the VEE WEAR
Preview: https://docs.microej.com/en/feature-ack/VEEWear/index.html

### CODE REVIEW

File: ApplicationDeveloperGuide/Android/android.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+MicroEJ provides a set of tools and libraries to run applications powered by MicroEJ on Android and Wear OS.
+This allows for the same application to be developed, simulated, tested, and run on MicroEJ VEE and Android alike.
+
+Having the same code ready for both Android and MicroEJ VEE opens up a wide range of use cases, including but not limited to:
+
+* Develop derivative products based on small MCUs or low-cost MPUs with limited resources where Android cannot be used as it is inherently resource-intensive.
+* Reduce energy consumption by enabling two processors to coexist to distribute tasks between a very powerful processor powered by Android and a low-power processor powered by MicroEJ.
+
+Below are some examples from the wearables segment that illustrate these use cases:
```
[the remark this answers is not in the document]
Comment (thomasponsmicroej) [reply ApplicationDeveloperGuide/Android/android.rst]: fixed

File: VEEWear/Android/overview.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Overview
+========
+
+The MicroEJ Android Compatibility Kit is composed of two main components:
+
+* A runtime: applications developed on MicroEJ can run on the Android platform thanks to the Android-based implementation of the MicroEJ Foundation libraries and dedicated support libraries.
+* A developer kit: the MicroEJ SDK 6 and a Gradle plugin provide the necessary support for developing applications in Android Studio using Gradle.
+
+
+Software Architecture
```
Comment (apoudret-microej) [inline VEEWear/Android/overview.rst:12]: restore ACK flow diagram

Comment (thomasponsmicroej) [reply VEEWear/Android/overview.rst:24]: can you check?

Comment (apoudret-microej) [reply VEEWear/Android/overview.rst:24]: ok great!

**Review by apoudret-microej — APPROVED** [review]

**Pull request merged**